src/rootPages/Designer/properties/process/ABProcessTaskServiceCalculate.js

```diff
@@ -0,0 +1,236 @@
+
+              insertFormula(component.symbol);
+
+              $$(ids.operatorPopup).hide();
+            },
+          },
+        },
+      });
+      let labelWidth = 120;
+      return {
+        rows: [
+          {
+            //     id: id,
+            view: "form",
+            elementsConfig: {
+              labelWidth: labelWidth,
+            },
+            elements: [
+              {
+                id: ids.name,
+                view: "text",
+                label: L("*Name"),
+                name: "name",
+                value: this.name,
+              },
+
+              {
+                id: ids.formulaText,
+                view: "texthighlight",
+                height: 200,
+                label: L("*Formula"),
+                type: "textarea",
+                value: this.formulaText || "",
+                highlight: (text) => {
+                  // list.forEach(function (item) {
+                  //   text = text.replace(
+                  //     new RegExp(`{${item.value}}`, "g"),
+                  //     `<span style='background: #90adb5; color:#000000;'>{${item.value}}</span>`
+                  //   );
+                  // });
+                  return text;
+                },
+              },
+              {
+                cols: [
+                  {
+                    width: labelWidth,
+                    fillspace: true,
+                  },
+                  {
+                    view: "button",
+                    css: "webix_primary",
+                    type: "icon",
+                    icon: "fa fa-at",
+                    label: L("*Parameters"),
+                    click: function () {
+                      // show popup
+                      $$(ids.variablePopup).show(this.$view);
+                    },
+                  },
+                  {
+                    view: "button",
+                    css: "webix_primary",
+                    type: "icon",
+                    icon: "fa fa-hashtag",
+                    label: L("*Operators"),
+                    click: function () {
+                      // show popup
+                      $$(ids.operatorPopup).show(this.$view);
+                    },
+                  },
+                ], //End Column
+              },
+            ], //End element
+          },
+        ], //End row
+      }; //End Return
+    } //End UI
+
+    populate(element) {
+      const ids = this.ids;
+
+      let list = (element?.process?.processDataFields(element) || []).map(
+        (item) => {
+          return {
+            id: item.key,
+            value: item.label,
+          };
+        }
+      );
+
+      $$(ids.name).setValue(element.name);
+      $$(ids.formulaText).setValue(element.formulaText);
+      $$(ids.variableList).define("data", list);
+      $$(ids.variableList).refresh();
+    }
+
+    values() {
+      var obj = {};
+      var ids = this.ids;
+
+      obj.label = $$(ids.name)?.getValue();
```

Comment:
should this be `obj.name`? on line 205 we set `$$(ids.name)` using the `name` property, but here we save it as `label`